packages/core/src/deployment/deployExecutionStateReducer.ts

```diff
@@ -79,17 +77,17 @@ function convertTo(
   vertexVisitResult: ExecutionVertexVisitResult
 ): VertexExecutionState {
   switch (vertexVisitResult._kind) {
-    case VertexResultEnum.SUCCESS:
+    case "success":
```

Comment:
What's the reasoning behind this change?

Reply:
I was under the impression that we prefer union types to enums, have I misunderstood that?

Is the issue that I have used the string literal instead of a `const SUCCESS = 'SUCCESS'`